### ISSUE TITLE
Pretrained weigths

### ISSUE THREAD

**Issue by abhigoku10** [ZhengPeng7/GLCNet#1]
@ZhengPeng7  hi thanks for the wonderful work and the code base , can you please share the pretrained weight file on google drive or on one drive 
Thanks in advance

**Comment by ZhengPeng7** [comment]
Okay, no problem, I'll share it on google-drive today. But you can still have a try to train, I think the performance is around the values in the paper since we didn't make modifications on the hyperparameters in modules like the CBGM or data augmentation.

**Comment by abhigoku10** [comment]
@ZhengPeng7  thanks for the response , thansk for sharing currently i donto have computing power so i want to test the model and check its accuracy then get approval for training so if you could share it will be helpful

**Comment by ZhengPeng7** [comment]
You can now check the links of weights in README I updated last night. The performance may be a little bit higher after some upgrades in the future, cuz some improvements still needs to be done on codes of cxt_psn branch.
